css-ui-3/Overview.bs

```diff
@@ -81,7 +81,6 @@ spec:css2; type:property; text:background-image
 <!-- Some of the following definitions have existing exported defintions from other specs,
 but not from specs that are OK to link to from a CR -->
 <pre class="anchors">
-url: https://www.w3.org/TR/CSS2/box.html#border-edge; type: dfn; spec: CSS2; text: border edge
```

Comment:
Same-ish comment as in css-contain: css-ui-3 is a REC, and if we need to publish an update, I'd rather not be tripped up by dependencies to a non REC, especially for no non-editorial gains.